### PR TITLE
fix rotundas used for etc., one dozen '2c' remaining to check visually in context

### PR DESCRIPTION
**Pull request by wollmers**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,8 @@
 *.png-
 *.txt
 
+*.bak
```
Comment (stweil) [inline .gitignore:7]: Backup file extensions (`*.bak`, `*~`, ...) and also other patterns like `.DS_Store` can be added to your personal Git settings, so there is no need to add them to project settings like this one. Ideally `.gitignore` should only ignore patterns which are required by the project.

File: gt/train/ONB_nfp_18730705_007/ONB_nfp_18730705_007.tif_tl_252.gt.txt [hunk 2]
```diff
@@ -1 +1 @@
-bedienſteten 2c., empfing das Ritterkrenz erſter Claſſe vom Orden
\ No newline at end of file
+bedienſteten ꝛc., empfing das Ritterkrenz erſter Claſſe vom Orden
```
Comment (stweil) [inline gt/train/ONB_nfp_18730705_007/ONB_nfp_18730705_007.tif_tl_252.gt.txt]: Looks like a transcription typo for `Ritterkreuz`.

File: gt/train/ONB_nfp_18950706_016/ONB_nfp_18950706_016.tif_tl_670.gt.txt [hunk 3]
```diff
@@ -1 +1 @@
-tikel der Textil⸗Brauche 2c. Mitinhaber
\ No newline at end of file
+tikel der Textil⸗Brauche ꝛc. Mitinhaber
```
Comment (stweil) [inline gt/train/ONB_nfp_18950706_016/ONB_nfp_18950706_016.tif_tl_670.gt.txt]: Branche?

File: gt/train/ONB_nfp_19110701_030/ONB_nfp_19110701_030.tif_tl_512.gt.txt [hunk 4]
```diff
@@ -1 +1 @@
-2c. 2c. kaufen Sie am vorteilhoftesten
\ No newline at end of file
+ꝛc. ꝛc. kaufen Sie am vorteilhoftesten
```
Comment (stweil) [inline gt/train/ONB_nfp_19110701_030/ONB_nfp_19110701_030.tif_tl_512.gt.txt]: Is it really "vorteilhoftesten"? Or "vorteilhaftesten"? Confirmed, so proposing fix.

Comment (stweil) [inline gt/train/ONB_nfp_19110701_030/ONB_nfp_19110701_030.tif_tl_512.gt.txt]:
```suggestion
ꝛc. ꝛc. kaufen Sie am vorteilhaftesten
```

File: gt/train/ONB_nfp_18730705_007/ONB_nfp_18730705_007.tif_tl_252.gt.txt (hunk 2 above)
Comment (stweil) [reply gt/train/ONB_nfp_18730705_007/ONB_nfp_18730705_007.tif_tl_252.gt.txt]:
```suggestion
bedienſteten ꝛc., empfing das Ritterkreuz erſter Claſſe vom Orden
```

File: gt/train/ONB_nfp_18950706_016/ONB_nfp_18950706_016.tif_tl_670.gt.txt (hunk 3 above)
Comment (stweil) [reply gt/train/ONB_nfp_18950706_016/ONB_nfp_18950706_016.tif_tl_670.gt.txt]:
```suggestion
tikel der Textil⸗Branche ꝛc. Mitinhaber
```

**Comment by stweil** [comment]
Thanks. The sample shows that we still have to expect lots of u/n confusions.

**Comment by wollmers** [comment]
I saw this u/n and some others during reviewing the diffs, but want to solve them either by dictionary check or optical/visual comparison. There are also (seldom) typesetting errors in the original image and it's questionable to overrule the typesetter. Ideally obvious typesetting errors should be tagged, like DTA (Deutsches Text Archiv) did it.

**Comment by stweil** [comment]
I absolutely agree. A dictionary check can only give a hint because of the typesetting errors in the original which must be preserved in the transcription.

**Pull request merged**